torch/testing/_internal/common_utils.py

```diff
@@ -170,19 +178,35 @@ def call_helper(self, *args):
 PY3 = sys.version_info > (3, 0)
 PY34 = sys.version_info >= (3, 4)
 
-def run_tests(argv=UNITTEST_ARGS):
-    if TEST_IN_SUBPROCESS:
-        suite = unittest.TestLoader().loadTestsFromModule(__main__)
-        test_cases = []
+def get_test_cases(suite_or_case):
```

Comment:
Perhaps name this function `discover_test_cases_recursively()`?  Or at least have a docstring that says that it works recursively.  It saves folks like me a few seconds to understand it.

Reply:
Also, how about this implementation:
```
def discover_test_cases_recursively(suite_or_case):

    if isinstance(suite_or_case, unittest.TestCase):
        return [suite_or_case]
    rc = []
    for element in suite_or_case:
        rc.extend(discover_test_cases_recursively(element))
    return rc
```

Reply:
@kostmo that was the original implementation, but I decided to avoid extra function call and call it in method body. But I guess perf for this one is not really important.